### PR TITLE
Radix Sort implemented

### PR DESCRIPTION
**Pull request by anasshamoon12002**
Radix Sort was not implemented, so I have implemented Radix Sort.

### CODE REVIEW

**Comment by williamfiset** [comment]
Thanks for the PR! Will review it this week :)

File: com/williamfiset/algorithms/sorting/RadixSort.java [hunk 1]
```diff
@@ -1,12 +1,73 @@
...
-package com.williamfiset.algorithms.sorting;
-
-public class RadixSort {
-
-  public static void radixSort(int[] ar) {
-    
-  }
-
-}
+
```
Comment (williamfiset) [inline com/williamfiset/algorithms/sorting/RadixSort.java]: Add package as the first line:

package com.williamfiset.algorithms.sorting;

File: com/williamfiset/algorithms/sorting/RadixSort.java [hunk 2]
```diff
@@ -1,12 +1,73 @@
...
+public class radixSort
```
Comment (williamfiset) [inline com/williamfiset/algorithms/sorting/RadixSort.java]: Capitalize radixSort class to be RadixSort

File: com/williamfiset/algorithms/sorting/RadixSort.java [hunk 3]
```diff
@@ -1,12 +1,73 @@
...
+        {
+
+            // Counting sort is also the part of radix sort. So, you should have the knowledge about it.
+
+            int b[] = new int[10];
+            int sorted[] = new int[a.length];
+            int max = a[0];
+            int divisor = 1;
+
+            for(int i = 0; i < sorted.length; i++)  // copy contents of a[] so that the original array is not changed
```
Comment (williamfiset) [inline com/williamfiset/algorithms/sorting/RadixSort.java]: Please avoid end line comments which create very long lines. Prefer placing the comment on its own line if it's too long.

File: com/williamfiset/algorithms/sorting/RadixSort.java (hunk 1 above)
Comment (williamfiset) [inline com/williamfiset/algorithms/sorting/RadixSort.java]: Can you add a few tests in a new file called at javatests/com/williamfiset/algorithms/sorting/RadixSortTest.java?
Make sure the new file has the package name javatests.com.williamfiset.algorithms.sorting and run your tests using gradle like "gradle test --tests javatests.com.williamfiset.algorithms.sorting.RadixSortTest". There are more instructions in the [README](https://github.com/williamfiset/Algorithms#adding-a-new-algorithm).

File: com/williamfiset/algorithms/sorting/RadixSort.java [hunk 4]
```diff
@@ -1,12 +1,73 @@
...
+        }
+
+
+        public static void main(String[] args)
+        {
+            int a[] = {10, 3, 1, 44, 100, 33, 3};
+
+            int sort[] = radix(a);
+
+            for(int i = 0; i < sort.length; i++)
```
Comment (williamfiset) [inline com/williamfiset/algorithms/sorting/RadixSort.java]: Tip: You can use java.util.Arrays.toString(sort); instead :)

File: com/williamfiset/algorithms/sorting/RadixSort.java [hunk 5]
```diff
@@ -1,12 +1,73 @@
...
+            int sorted[] = new int[a.length];
+            int max = a[0];
+            int divisor = 1;
+
+            for(int i = 0; i < sorted.length; i++)  // copy contents of a[] so that the original array is not changed
+            {
+                sorted[i] = a[i];
+            }
+
+            for(int i = 0; i < sorted.length; i++)  // finding max to implement counting sort. (Range of elements)
```
Comment (williamfiset) [inline com/williamfiset/algorithms/sorting/RadixSort.java]: I have not yet looked at the logic behind the algorithm -- waiting until you add tests.

File: com/williamfiset/algorithms/sorting/RadixSort.java (hunk 1 above)
Comment (anasshamoon12002) [reply com/williamfiset/algorithms/sorting/RadixSort.java]: Ok. I'll add that.

Comment (anasshamoon12002) [reply com/williamfiset/algorithms/sorting/RadixSort.java]: @williamfiset 
I am using JUnit4 testing, is it okay? I don't particularly know how to use gradle? Is that okay.
Please, guide me.

**Pull request closed without merging**